types/cookie_js/index.d.ts

```diff
@@ -17,6 +17,23 @@ declare module 'cookiejs' {
 declare function cookie(key : string, fallback?: string) : string;
 declare function cookie(keys : string[], fallback?: string) : string;
 
+interface UtilStatic {
+  isArray: (value: any) => boolean;
+  isObject: (value: any) => boolean;
+  toArray: (...args: any[]) => any[];
+  getKeys: (value: {}) => any[];
+  encode: (value: string) => void;
```

Comment:
As I see here https://github.com/florian/cookie.js/blob/deaad9737e61c82a8b6e50053620cc426d645662/cookie.js#L39 `encode` and `decode` functions aren't `void`. Am I wrong?

Reply:
@slawiko 

Thanks for the review! 🙇 You're definitely correct. 
I squshed the changes!